src/pipx/main.py

```diff
@@ -528,6 +549,13 @@ def _add_reinstall(subparsers, venv_completer: VenvCompleter, shared_parser: arg
             f"Requires Python {MINIMUM_PYTHON_VERSION} or above."
         ),
     )
+    p.add_argument(
```

Comment:
Let's create a helper function in the style of `add_include_dependencies`, instead of duplicating?

Reply:
not sure what that would look like... could you provide an example implementation?

Reply:
I basically suggest adding this option where needed like in this line:

https://github.com/pypa/pipx/blob/e28c7b1033eded0242d7b5d37156a8e8f94a49ba/src/pipx/main.py#L348

Only with a different underlying function.

Reply:
done!